src/main/java/teammates/common/datatransfer/attributes/InstructorAttributes.java

```diff
@@ -17,6 +18,15 @@
 
     public static final String DEFAULT_DISPLAY_NAME = "Instructor";
 
+    /**
+    * Sort the Instructors list alphabetically by name.
+    */
```

Comment:
Fix the javadoc formatting - these two lines need to be indented one more space to the right such that the asterisks align.